--- service.py.orig	2017-11-20 03:01:47.125161482 +0000
+++ service.py	2017-11-20 03:02:28.213655231 +0000
@@ -412,8 +412,9 @@
         admin._create_admin_user(context, os_admin,
                                  encrypt_password=True)
 
-        PgSqlAdmin(os_admin).alter_user(context, postgres, None,
-                                        'NOSUPERUSER', 'NOLOGIN')
+        # TODO: (andybotting) This is required for PostgreSQL startup on Ubuntu
+        #PgSqlAdmin(os_admin).alter_user(context, postgres, None,
+        #                                'NOSUPERUSER', 'NOLOGIN')
 
         self.set_current_admin_user(os_admin)
